src/components/mainView.tsx

```diff
@@ -31,7 +32,7 @@ function MainView() {
       </Stack>
       <StudentSummary />
       <StudentDetails />
-      <InputModal visible={show}  setVisible={setShow}/>
+      <StudentInputModal isEdit={false} visible={show}  setVisible={setShow}/>
```

Comment:
`isEdit` is not required as the edit modal will require either the student data or id. So if they exist modal type will be for edit.